test/vast/Dialect/HighLevel/asm-b.c

```diff
@@ -0,0 +1,12 @@
+// RUN: %vast-front -vast-emit-mlir=hl %s -o - | %file-check %s
+// RUN: %vast-front -vast-emit-mlir=hl %s -o %t && %vast-opt %t | diff -B %t -
+
+void f() {
+int src = 1;
+int dst;
+// CHECK: hl.inline_asm {is_volatile} #core.strlit<"mov %1, %0\n\tadd $1, %0">([0] %2 : [#core.strlit<"=r">]) (ins : [1] %4 : [#core.strlit<"r">]) () () : (!hl.lvalue<!hl.int>, !hl.int) -> ()
```

Comment:
can you add #core.strlit alias to make this less verbose?

Reply:
I have tried and struggled with outputing "special" symbols via `llvm::raw_ostream`. They are show as their ascii codes instead.